include/uriparser/Uri.h

```diff
@@ -201,6 +201,15 @@ typedef struct URI_TYPE(QueryListStruct) {
 } URI_TYPE(QueryList); /**< @copydoc UriQueryListStructA */
 
 
+/**
+ * Checks if a URI has the host component set.
+ *
+ * @param uri %URI to check
+ * @return <c>URI_TRUE</c> when host is set, <c>URI_FAlSE</c> otherwise
+ */
+URI_PUBLIC UriBool URI_FUNC(IsHostSet)(const URI_TYPE(Uri) * uri);
```

Comment:
I'm considering a rename from `IsHostSet` to `HasHost`. What do you think?

Reply:
I'm fine with whatever name you prefer, both make sense to me. Do you want me to  rename the function? Do you want two commits, or one is also fine (and then the commit message would be different)?

Reply:
@kocsismate two commits would be ideal: one for making public, one for the rename. Thanks!